client/pages/login.vue

```diff
@@ -10,6 +10,9 @@ const usernamePlaceholder = getRandomDeveloperName()
 const message = ref<Nullable<string>>(null)
 const messageSeverity = ref<'success' | 'info' | 'warn' | 'error' | undefined>('error')
 
+const usernameField = ref()
+const passwordField = ref()
+
```

Comment:
I can't add this comment on the right segment because github won't let me do it...

but, line 34, errors is now unused